static/include/components/board/board-analogs.md

```diff
@@ -72,6 +66,6 @@ The following properties are available for `analogs`:
 |`name` | string | **Required** | Your name for the analog reader. |
 |`pin`| string | **Required** | The pin number of the ADC's connection pin, wired to the board. This should be labeled as the physical index of the pin on the ADC.
 |`chip_select`| string | **Required** | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board's connection pin, wired to the ADC. |
```

Comment:
Similarly, this has changed on rpi4's from the physical pin number to the chip select index. On most boards, if you're using physical pin 24, set it to 0, and if you're using physical pin 26, set it to 1. On the raspberry pi 4, I think there are extra SPI buses you can enable by changing the devicetree overlays in /boot/config.txt, but I haven't looked into that myself.

For every board that's not a raspberry pi, we've already been using the chip select values, and it's been either 0 or 1 the entire time.